doc/SPEC.md

```diff
@@ -80,7 +80,7 @@ class RightTriangle extends XElement {
 
   static template(html) {
     return ({ base, height, hypotenuse }) => html`
-      <code>Math.hypot(${base}, ${height}) = ${hypotenuse}<code>
+      <code>Math.hypot(${base}, ${height}) = ${hypotenuse}</code>
```

Comment:
Noticed some mistakes in this file. Not related to interface changes, but we should fix anyhow.